src/services/api/savedFilter/models.ts

```diff
@@ -8,8 +8,8 @@ export type TUserSavedFilter = ISavedFilter & {
 };
 
 export enum SavedFilterTag {
-  PARTICIPANT = 'participant',
-  VARIANT = 'variants-exploration',
+  PARTICIPANTS_DExP = 'participants-data-exploration-page',
```

Comment:
Is DexP volontary here ?

Reply:
Yes, for `Data Explore Page`. In case we need we need to support participants filter elsewhere in the app.

Reply:
Should our enum be un `PascalCase`? https://www.typescriptlang.org/docs/handbook/enums.html